model/Resource/ServiceProvider/ResourceServiceProvider.php

```diff
@@ -46,5 +50,24 @@ public function __invoke(ContainerConfigurator $configurator): void
                     service(QueueDispatcherInterface::SERVICE_ID),
                 ]
             )->public();
+
+        $services->set(IndexDocumentBuilderFactory::class, IndexDocumentBuilderFactory::class)
+            ->private();
+
+        $services->set(IndexDocumentBuilder::class)
+            ->factory(
+                [
+                    service(IndexDocumentBuilderFactory::class),
+                    'create'
+                ]
+            )->private();
+
+        $services->set(RdfMediaRelationRepository::class, RdfMediaRelationRepository::class)
+            ->factory(
+                [
+                    RdfMediaRelationRepositoryFactory::class,
+                    'getRdfMediaRelationRepository'
+                ]
+            )->private();
```

Comment:
- [ ] All can be deleted, right?

```suggestion
```

Reply:
This was already deleted (and all usages now fetch it from the `ServiceLocator` instead), so marking this thread as solved.